<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SqleditorDialog</class>
 <widget class="QDialog" name="SqleditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Query editor</string>
  </property>
  <property name="windowIcon">
   <iconset theme="drive-multidisk">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="7">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="fsplitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QListView" name="tabsView">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::DragOnly</enum>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::IgnoreAction</enum>
       </property>
      </widget>
      <widget class="QListView" name="fieldsView">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::DragOnly</enum>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::IgnoreAction</enum>
       </property>
      </widget>
     </widget>
     <widget class="QSplitter" name="rsplitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QPlainTextEdit" name="queryEdit">
       <property name="placeholderText">
        <string>Type a query or select table from list and add &quot;where&quot; keyword to filter results</string>
       </property>
      </widget>
      <widget class="QTableView" name="tableView">
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
      </widget>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QComboBox" name="helperCombo"/>
       </item>
       <item>
        <widget class="QListView" name="helperView">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="runButton">
     <property name="text">
      <string>Run query</string>
     </property>
     <property name="icon">
      <iconset theme="system-run">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="addButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Add record</string>
     </property>
     <property name="icon">
      <iconset theme="list-add">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QToolButton" name="delButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Delete record</string>
     </property>
     <property name="icon">
      <iconset theme="list-remove">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QToolButton" name="saveButton">
     <property name="text">
      <string>Commit</string>
     </property>
     <property name="icon">
      <iconset theme="document-save">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QToolButton" name="undoButton">
     <property name="text">
      <string>Rollback</string>
     </property>
     <property name="icon">
      <iconset theme="edit-undo">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>337</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="1">
    <widget class="QToolButton" name="exitButton">
     <property name="text">
      <string>Exit</string>
     </property>
     <property name="icon">
      <iconset theme="application-exit">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabsView</tabstop>
  <tabstop>queryEdit</tabstop>
  <tabstop>fieldsView</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>runButton</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>delButton</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>undoButton</tabstop>
  <tabstop>exitButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
